tests/units/cli/test_config.py

```diff
@@ -300,6 +300,9 @@ def test_save_config(self):
                 }
             },
             'temp_dir': './pipelinewise-test-config/tmp',
+            'tap_id': 'mysql_sample',
+            'query_tag':
```

Comment:
can you create another test_config and tests where the `query_tag` is not present?
This is to make sure the ppw and all component are backward compatible.

Reply:
I can't create another test_config because it is generated and the expected output is to always include `tap_id` and `query_tag` in the generated config.

To make sure it's still compatible with old config which doesn't have these properties, please check the next point.

Reply:
Got it